app/controllers/middleware_server_controller.rb

```diff
@@ -110,17 +111,28 @@ def add_jdbc_driver
       :driver_minor_version => params["minorVersion"]
     }
 
-    run_server_operation(STANDALONE_SERVER_OPERATIONS.fetch(:middleware_add_jdbc_driver), selected_server)
-    render :json => {
-      :status => :success, :msg => _("JDBC Driver \"%s\" has been installed on this server.") % params["driverName"]
-    }
+    op_trigger = run_server_operation(STANDALONE_SERVER_OPERATIONS.fetch(:middleware_add_jdbc_driver), selected_server)
+    if op_trigger
+      flash_param_dict = { :driverName => params["driverName"]}
+      if params[:id]
+        add_flash(_("JDBC Driver \"%{driverName}\" has been installed on this server.") \
+                  % flash_param_dict, :success)
+      else
+        add_flash(_("JDBC Driver \"%{driverName}\" has been installed on selected servers.") \
+                  % flash_param_dict, :success)
+      end
+    end
+    javascript_flash
   end
 
   def jdbc_drivers
-    mw_server = MiddlewareServer.find(from_cid(params[:server_id]))
+    mw_servers = params[:server_id]
+    # Get the feed of the servers, in case of multiple servers we
+    # can assume that all of them are on the same feed, so we get the first one.
```

Comment:
Your provided screenshots, suggest that you can select servers living in different feeds. So, a big question mark :interrobang: landed over my head when I saw this assumption.
When I saw the JS, I was expecting this method to return drivers common across all servers.
Does this means that the UI doesn't let you select servers living in different feeds?